gui-wm/hyprland/files/nvidia-0.27.2.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/render/gles2/renderer.c b/render/gles2/renderer.c
```

Comment:
It would be good to have patch description including links to the gentoo bug, hyrpland pull request and if you got this patch from somewhere else.

https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#patch-descriptions